index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Oluyemi Portfolio</title>
+    <!-- CSS only -->
+    <script src="https://kit.fontawesome.com/746ba03926.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+    <header>
+        <nav id="navbar">
+            <div id="profileName">
+                <p><strong>Paul, Oluyemi</strong></p>
+            </div>
+            <div id="bars">
+                <div class="bar1"></div>
+                <div class="bar2"></div>
+                <div class="bar3"></div>
+            </div>
+        </nav>
+    </header>
+    <main id="container">
+        <header id="greetText">
+            <h1>Hey There.<br>
+                I'm Oluyemi</h1>
+        </header>
+        <section>
+            <h2 id="jobTitle">I am a Software Developer</h2>
+            <p id="description">I can help you build a product , feature or website Look through
```

Comment:
- [ ] It's always a best practice to always use the class attribute for multiple elements (do not use ID selector  for multiple elements), kindly change your id into the class, you can visit this [link](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) for more information